torch_geometric/data/data.py

```diff
@@ -348,7 +364,10 @@ class Data(BaseData):
     def __init__(self, x: OptTensor = None, edge_index: OptTensor = None,
                  edge_attr: OptTensor = None, y: OptTensor = None,
                  pos: OptTensor = None, **kwargs):
-        super().__init__()
+        # `Data` doesn't support group_name, so we need to adjust `TensorAttr`
+        # accordingly here to avoid requiring `group_name` to be set:
+        super(Data, self).__init__(attr_cls=DataTensorAttr)
```

Comment:
```suggestion
        super().__init__(attr_cls=DataTensorAttr)
```
should work as well?